.workflows-labels.yaml

```diff
@@ -0,0 +1,11 @@
+critical:
```

Comment:
suggest a more descriptive name for this file

Maybe: `default_rule_set_for_workflow_labels_and_annotations.yaml`?